submissions/TatyanaMolchanova/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,174 @@
+                </a>
+
+                <div id="myhtmlpopup" class="overlay">
+                    <div class="popup">
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/apps.png" alt>
+                                Apps
+                           </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/chat.png" alt>
+                                Chat
+                           </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/blogs.png" alt>
+                                Blogs
+                            </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/calendar.png" alt>
+                                Calendar
+                            </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/classroom.png" alt>
+                                Class
+                            </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/files.png" alt>
+                                Files
+                            </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/mail.png" alt>
+                                Mail
+                            </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/maps.png" alt>
+                                Maps
+                            </a>
+                        </div>
+                        <div class="popup-item">
+                           <a href="#" class="popup-link">
+                                <img src="./images/mars.png" alt>
+                                Mars
+                            </a>
+                        </div>
+                    </div>
+                    
+                    <div class="more-wrap">
+                        <div style="text-align: center;">
+                            
+                            <input type="checkbox" class="read-more-state" id="post-1" />
```

Comment:
Button 'More`  without focus: I can't access this element using the keyboard. See video

Reply:
I fixed it.